src/utils/ASTHelpers.ts

```
@@ -115,6 +115,8 @@ export class ASTHelpers {
           this.declarationIncludesIdentifier(node.consequent) ||
           this.declarationIncludesIdentifier(node.alternate)
         );
+      case 'TemplateLiteral':
+        return node.expressions.some((expr) => this.declarationIncludesIdentifier(expr));
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)